pkg/notes/notes_map.go

```diff
@@ -24,9 +24,9 @@ import (
 	"path/filepath"
 
 	"github.com/sirupsen/logrus"
-	"gopkg.in/yaml.v2"
 
 	"sigs.k8s.io/release-sdk/object"
+	yaml "sigs.k8s.io/yaml/goyaml.v2"
```

Comment:
@cpanato this uses NewDecoder which is not available in ghodds/yaml / sigs.k8s.io/yaml